microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -122,7 +122,7 @@ On the **Properties** webpage, enter the following information to specify proper
 |:--- |:--- |
 | Category (required) | The category that best describes your extension.  Listing your extension in the right category helps users find your extension easily and understand more about it.  |
 | Privacy policy requirements (required) | Indicate if your extension accesses, collects, or transmits any personal information.  Your extension might fail the certification step if you select **Yes** and you don't provide a `Privacy policy URL`. |
-| Privacy policy URL | A valid privacy policy URL to communicate how your extension follows privacy laws and regulations.  You're responsible for ensuring your extension follows privacy laws and regulations.  You're also responsible for providing a privacy policy URL if any personal information is being accessed, transmitted, or collected by your extension.  To determine if your extension requires a privacy policy, see [Microsoft Edge Developer Agreement](/legal/windows/agreements/app-developer-agreement) and [Developer policies for the Microsoft Edge Add-ons store](../store-policies/developer-policies.md). |
+| Privacy policy URL | A valid privacy policy URL to communicate how your extension follows privacy laws and regulations.  You're responsible for ensuring your extension follows privacy laws and regulations.  You're also responsible for providing a privacy policy URL if any personal information is being accessed, transmitted, or collected by your extension.  To determine if your extension requires a privacy policy, see [App Developer Agreement](https://go.microsoft.com/fwlink/p/?LinkID=221922) and [Developer policies for the Microsoft Edge Add-ons store](../store-policies/developer-policies.md). |
```

Comment:
fyi to test old link either navigate to it in rendered page of PR, or prepend the domain to the path.

old link tagged as /legal/windows/agreements/app-developer-agreement
prepend the domain:
https://learn.microsoft.com/legal/windows/agreements/app-developer-agreement

Reply:
@captainbrosset - this PR normalizes how link to App Dev Agreement: use fwlink, not the resulting redirected url.

Reply:
moved this "how to test old link" info to top of PR main page